### PR TITLE
Clean up old payment related code

### PR DESCRIPTION
**Pull request by cihangir**
Removes any code that touches old payment system, includes:
- Koding provider and its tests
  - Checkers
  - Snapshots
  - Fetchers
  - Cleaners
  - Plans
  - Solo to team migrator
- Clean up `kloud/queue`  
- Configuration options
- Webhook worker
- Payment nodejs handlers
- Paypal Go package

~~depends on https://github.com/koding/koding/pull/8882~~

### CODE REVIEW

**Comment by cihangir** [comment]
updated PR of https://github.com/koding/koding/pull/8951

**Review by rjeczalik — CHANGES_REQUESTED** [review]
Looks like broken env.sh gets generated and because of it `check connectivity` step is failing.

```
/opt/koding/client/../run exec /opt/koding/client/../scripts/generate-bongo-schema > /opt/koding/client/.schema.json
/opt/koding/client/../.env.sh: line 464: unexpected EOF while looking for matching `''
/opt/koding/client/../.env.sh: line 784: syntax error: unexpected end of file
error: could not parse KONFIG_JSON environment variable
```

**Comment by cihangir** [comment]
echoip.net ( or dreamhost.com ) is down

https://cloudup.com/cpRCC6v0lJ5

**Comment by cihangir** [comment]
changed endpoint to use ours https://github.com/koding/koding/pull/9042/commits/c901802a507cf872727a88b9d249931102a98141

File: configure [hunk 1]
```diff
@@ -162,7 +162,7 @@ git.long (str) ->
     updateVault()
   else
     log "lemme find your IP"
-    request "http://echoip.net", (err, res, body) ->
+    request "https://p.koding.com/-/ip", (err, res, body) ->
```
Comment (cihangir) [inline configure]: cc/ @szkl

**Comment by rjeczalik** [comment]
Failed on -race in klient/client tests, most likely the map here needs a mutex:

https://github.com/koding/koding/blob/master/go/src/koding/klient/client/client_test.go#L331

It's a test code, so it's harmless, restarted.

File: configure (hunk 1 above)
Comment (szkl) [inline configure]: I have cherry picked this into `master`.

**Review by rjeczalik — APPROVED** [review]

**Comment by rjeczalik** [comment]
@cihangir 👍

**Pull request merged**